api/client/commands.go

```diff
@@ -222,6 +251,9 @@ func (cli *DockerCli) CmdBuild(args ...string) error {
 	if *pull {
 		v.Set("pull", "1")
 	}
+
+	v.Set("f", *dockerfileName)
```

Comment:
ah ok but still the question is whether the API should match the CLI. Doesn't it usually?